odoo/addons/test_read_group/tests/test_group_expand.py

```diff
@@ -87,3 +87,111 @@ def test_full(self):
                 '__domain': [('state', '=', False)],
             },
         ])
+
+
+@common.tagged("test_read_group_selection")
+class TestSelectionReadGroup(common.TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.Model = self.env['test_read_group.on_selection']
+
+    def test_selection_read_group_order(self):
```

Comment:
You could combine this test method and the next one by having records using 2 out of 3 of the values.  The test method would check both the groups order and empty groups.

Reply:
yes, but then it is not a "unit" test 😜